ProcessLib/TES/TESProcess.cpp

```diff
@@ -379,15 +379,15 @@ TESProcess::computeRelativeHumidity(
          dof_table_single.dofSizeWithoutGhosts(),
          &dof_table_single.getGhostIndices(), nullptr});
 
-    GlobalIndexType nnodes = this->_mesh.getNumberOfNodes();
+    GlobalIndexType nnodes = _mesh.getNumberOfNodes();
 
     for (GlobalIndexType node_id = 0; node_id < nnodes; ++node_id)
     {
-        auto const p = getNodalValue(x, this->_mesh, dof_table, node_id,
+        auto const p = NumLib::getNodalValue(x, _mesh, dof_table, node_id,
                                      COMPONENT_ID_PRESSURE);
-        auto const T = getNodalValue(x, this->_mesh, dof_table, node_id,
+        auto const T = NumLib::getNodalValue(x, _mesh, dof_table, node_id,
                                      COMPONENT_ID_TEMPERATURE);
-        auto const x_mV = getNodalValue(x, this->_mesh, dof_table, node_id,
+        auto const x_mV = NumLib::getNodalValue(x, _mesh, dof_table, node_id,
                                         COMPONENT_ID_MASS_FRACTION);
 
```

Comment:
Have a look in TESProcess::computeEquilibriumLoading; `NumLib::` and `this->` are still used there...